### DISCUSSION TITLE
图片标题加粗

### DISCUSSION THREAD

**Discussion by fantasyEthan** [nju-lug/NJUThesis#139] [Q&A]
同学提醒发现的，
<img width="312" alt="image" src="https://user-images.githubusercontent.com/54493995/167101950-8b13402e-871a-4283-ac24-d80733e6fd6f.png">
<img width="478" alt="image" src="https://user-images.githubusercontent.com/54493995/167102001-b927d623-c5cd-4538-b34d-b4c74b3056c9.png">
当然可以直接在caption内部加粗，不过我已经都写完了，想问问有没有统一的接口去调整或者写个宏去改?

**Comment by atxy-blip** [comment]
导言区加下面两条命令

```TeX
\captionsetup[figure]{textfont=bf}
\captionsetup[table]{textfont=bf}
```
